packages/cli/subtasks/load-deployment.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
```

Comment:
This could have been done in a separate PR. In fact, we should've made an issue for it. But, if it's not too chaotic, we can include it here.

Basically, what this does is fetch the target deployment data and inject it at runtime. We could make this something that the deployer plugin can do, for other plugins to use, and not have every plugin implement it.

Reply:
Actually yeah, I moved this subtask to the deployer.

Even though the deployer does not use it, other plugins (campatible with the deployer) can use it.